✨ [source-surveyCTO] Add New streams

d-kimanthi opened the pull request:
## Fix: Add new streams to SurveyCTO: `FormData`, `FormDataset`, `Mediafiles`, `FormDefinitionData`, `FormRepeatGroupData`

## Context
The current existing SurveyCTO source connector has only one stream that pulls form data from SurveyCTO. However, sometimes the forms in SurveyCTO have more data and metadata that might be needed. Some of this extra data includes: survey form metadata, media files, datasets and repeat groups. This PR adds new streams that allow the connector to pull this extra form information whenever needed.

## Changes
We have made changes to the `streams.py` and `Helpers.py` file to have a total of 5 streams named the following:

-  `FormData` - This stream handles all the form data from the survey. This is the only functionality that exists in the current version of the connector
- `FormDataset` - handles getting data from a dataset.
- `Mediafiles` - handles getting urls of the media files and downloading them.
- `FormDefinitionData` - this stream gets all the metadata of the survey
- `FormRepeatGroupData` - stream handles extracting repeat groups data for a given SurveyCTO form id

Commit:
🏥Source Amplitude: fix CAT (#35743)

tolik0 committed:
:bug: Source Google Ads: Fix error for new customers for incremental events streams (#35664)

Commit:
🏥Source Amplitude: fix formatting (#35748)

tolik0 committed:
✨ Source Microsoft OneDrive: Enable connector in Cloud (#35584)

Commit:
🏥Source Chargebee: fix CAT (#35750)

ChristoGrab committed:
Omit run.py from test coverage for source Sendgrid, Sentry, Zendesk Chat (#35728)

evantahler committed:
remove archived connectors code (#35689)

Commit:
🏥Source Twilio: fix CAT (#35751)

Commit:
🏥Source Sendgrid: fix CAT (#35753)

evantahler committed:
Archive connectors already hidden with legacy state (#35752)

evantahler committed:
Archive the code of 8 sources with legacy state (#35757)

evantahler committed:
source-recurly is archived - remove from OSS registry

gisripa committed:
Use a separate concurrency group for Java CDK GH workflow. (#35758)

edgao committed:
DV2 destinations: Build DestinationState / Migration framework (#35303)

Signed-off-by: Gireesh Sreepathi <gisripa@gmail.com>
Co-authored-by: Gireesh Sreepathi <gisripa@gmail.com>

edgao committed:
Java CDK: Staging destinations include timezone in extracted_at in CSV file (#35313)

Co-authored-by: Gireesh Sreepathi <gisripa@gmail.com>

jbfbell committed:
Certify Postgres (#35760)

jbfbell committed:
Postgres Strict Encrypt (#35762)

stephane-airbyte committed:
add a timeout to junit test (#35767)

Today there's no clear timeout for tests. The only one that exists is at the github action level, and it has to be conservative at 1h.
This change introduces a timeout per test method. The default value is set at 5 minutes per method, and can be overriden on a per method basis via the use of the junit @Timeout annotation.

We're also fixing a regression in the test ContainerFactory that was sharing less container than was possible. Finally, we're introducing a MdcScope.doNothing to allow using MdcScope in try blocks without triggering a compiler warning

alafanechere committed:
airbyte-ci format: run a runner with more disk space (#35790)

Commit:
🏥Source Paypal Transaction: fix CAT (#35781)

Commit:
🏥Source Google Search Console: fix expected records (#35787)

Commit:
🏥Source Zendesk Talk: change order of authentication methods in spec (#35783)

artem1205 committed:
🐛 Source SalesForce: fix memory leak (OOM) (#35791)

Signed-off-by: Artem Inzhyyants <artem.inzhyyants@gmail.com>

Commit:
🏥Source Pinterest: fix CAT (#35788)

ChristoGrab committed:
Source Recurly: Unarchive and release as 1.0.0 (#35763)

Commit:
airbyte-ci: add kotlin support, update license message template (#35784)

edgao committed:
Destination bigquery: upgrade cdk (#35315)

Signed-off-by: Gireesh Sreepathi <gisripa@gmail.com>
Co-authored-by: Gireesh Sreepathi <gisripa@gmail.com>

stephane-airbyte committed:
minor changes to the java CDK test code (#35774)

 

---

<details open="true"><summary>Generated summary (powered by <a href="https://app.graphite.dev">Graphite</a>)</summary>

> ## TL;DR
> This pull request adds logging for non-public JUnit methods and updates test methods to be public.
> 
> ## What changed
> - Added logging for non-public JUnit methods in `LoggingInvocationInterceptor.java`.
> - Updated test methods to be public in `CdcSourceTest.java`.
> - Updated logging message in `DefaultAirbyteSource.java`.
> 
> ## How to test
> No specific testing instructions provided.
> 
> ## Why make this change
> - To improve visibility and debugging by logging non-public JUnit methods.
> - To ensure consistency and best practices by making test methods public.
</details>

brianjlai committed:
Emit final state message for full refresh syncs and consolidate read flows (#35622)

Commit:
Fix expected records

ChristoGrab committed:
Source Recurly: update support level metadata (#35815)

evantahler committed:
Archive `destination-vertica`, restore `destination-vectara` (#35806)

erohmensing committed:
file cdk: fix typing, pull out non-scalar handling (#35687)

<!--
Thanks for your contribution! 
Before you submit the pull request, 
I'd like to kindly remind you to take a moment and read through our guidelines
to ensure that your contribution aligns with the type of contributions our project accepts.
All the information you need can be found here:
   https://docs.airbyte.com/contributing-to-airbyte/

We truly appreciate your interest in contributing to Airbyte,
and we're excited to see what you have to offer! 

If you have any questions or need any assistance, feel free to reach out in #contributions Slack channel.
-->

## What
* Fix typing and handling of different types in `_to_output_value` - we don't always get a `Scalar`. We already handle the different cases correctly, but the typing doesn't reflect this. 
* Splitting out the methods to do the scalar separately is a helpful precursor to https://github.com/airbytehq/airbyte/pull/35688, as the `DictionaryArray` object doesn't have an `as_py()` method.

## 🚨 User Impact 🚨
None

## Pre-merge Actions
*Expand the relevant checklist and delete the others.*

<details><summary><strong>New Connector</strong></summary>

### Community member or Airbyter

- **Community member?** Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- Connector version is set to `0.0.1`
    - `Dockerfile` has version `0.0.1`
- Documentation updated
    - Connector's `README.md`
    - Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - `docs/integrations/<source or destination>/<name>.md` including changelog with an entry for the initial version. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
    - `docs/integrations/README.md`

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- Create a non-forked branch based on this PR and test the below items on it
- Build is successful
- If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).

</details>

<details><summary><strong>Updating a connector</strong></summary>

### Community member or Airbyter

- Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- Unit & integration tests added


### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- Create a non-forked branch based on this PR and test the below items on it
- Build is successful
- If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).

</details>

<details><summary><strong>Connector Generator</strong></summary>

- Issue acceptance criteria met
- PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/resources/pull-requests-handbook)
- If adding a new generator, add it to the [list of scaffold modules being tested](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connector-templates/generator/build.gradle#L41)
- The generator test modules (all connectors with `-scaffold` in their name) have been updated with the latest scaffold by running `./gradlew :airbyte-integrations:connector-templates:generator:generateScaffolds` then checking in your changes
- Documentation which references the generator is updated as needed

</details>

<details><summary><strong>Updating the Python CDK</strong></summary>

### Airbyter

Before merging:
- Pull Request description explains what problem it is solving
- Code change is unit tested
- Build and my-py check pass
- Smoke test the change on at least one affected connector
   - On Github: Run [this workflow](https://github.com/airbytehq/airbyte/actions/workflows/connectors_tests.yml), passing `--use-local-cdk --name=source-<connector>` as options
   - Locally: `airbyte-ci connectors --use-local-cdk --name=source-<connector> test`
- PR is reviewed and approved
      
After merging:
- [Publish the CDK](https://github.com/airbytehq/airbyte/actions/workflows/publish-cdk-command-manually.yml)
   - The CDK does not follow proper semantic versioning. Choose minor if this the change has significant user impact or is a breaking change. Choose patch otherwise.
   - Write a thoughtful changelog message so we know what was updated.
- Merge the platform PR that was auto-created for updating the Connector Builder's CDK version
   - This step is optional if the change does not affect the connector builder or declarative connectors.

</details>

erohmensing committed:
file cdk: handle scalar values that resolve to None (#35688)

<!--
Thanks for your contribution! 
Before you submit the pull request, 
I'd like to kindly remind you to take a moment and read through our guidelines
to ensure that your contribution aligns with the type of contributions our project accepts.
All the information you need can be found here:
   https://docs.airbyte.com/contributing-to-airbyte/

We truly appreciate your interest in contributing to Airbyte,
and we're excited to see what you have to offer! 

If you have any questions or need any assistance, feel free to reach out in #contributions Slack channel.
-->

## What
* Closes https://github.com/airbytehq/airbyte/issues/34151
* Closes https://github.com/airbytehq/oncall/issues/4386

## How
Handle cases where the python value of a pyarrow scalar is None. This can be due to null values in data, as well as null-like values like `NaT` (similar to `NaN`). We previously handled this for `None` binary types, but now handle this for `None` of any type.

## 🚨 User Impact 🚨
No breaking changes. After this CDK version is released we should update the CDK dependency in S3 and any other file sources that parse parquet


## Pre-merge Actions
*Expand the relevant checklist and delete the others.*

<details><summary><strong>New Connector</strong></summary>

### Community member or Airbyter

- **Community member?** Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- Connector version is set to `0.0.1`
    - `Dockerfile` has version `0.0.1`
- Documentation updated
    - Connector's `README.md`
    - Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - `docs/integrations/<source or destination>/<name>.md` including changelog with an entry for the initial version. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
    - `docs/integrations/README.md`

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- Create a non-forked branch based on this PR and test the below items on it
- Build is successful
- If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).

</details>

<details><summary><strong>Updating a connector</strong></summary>

### Community member or Airbyter

- Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- Unit & integration tests added


### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- Create a non-forked branch based on this PR and test the below items on it
- Build is successful
- If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).

</details>

<details><summary><strong>Connector Generator</strong></summary>

- Issue acceptance criteria met
- PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/resources/pull-requests-handbook)
- If adding a new generator, add it to the [list of scaffold modules being tested](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connector-templates/generator/build.gradle#L41)
- The generator test modules (all connectors with `-scaffold` in their name) have been updated with the latest scaffold by running `./gradlew :airbyte-integrations:connector-templates:generator:generateScaffolds` then checking in your changes
- Documentation which references the generator is updated as needed

</details>

<details><summary><strong>Updating the Python CDK</strong></summary>

### Airbyter

Before merging:
- Pull Request description explains what problem it is solving
- Code change is unit tested
- Build and my-py check pass
- Smoke test the change on at least one affected connector
   - On Github: Run [this workflow](https://github.com/airbytehq/airbyte/actions/workflows/connectors_tests.yml), passing `--use-local-cdk --name=source-<connector>` as options
   - Locally: `airbyte-ci connectors --use-local-cdk --name=source-<connector> test`
- PR is reviewed and approved
      
After merging:
- [Publish the CDK](https://github.com/airbytehq/airbyte/actions/workflows/publish-cdk-command-manually.yml)
   - The CDK does not follow proper semantic versioning. Choose minor if this the change has significant user impact or is a breaking change. Choose patch otherwise.
   - Write a thoughtful changelog message so we know what was updated.
- Merge the platform PR that was auto-created for updating the Connector Builder's CDK version
   - This step is optional if the change does not affect the connector builder or declarative connectors.

</details>

edgao committed:
Destination redshift: Upgrade cdk (#35316)

Signed-off-by: Gireesh Sreepathi <gisripa@gmail.com>
Co-authored-by: Gireesh Sreepathi <gisripa@gmail.com>

erohmensing committed:
🤖 Bump patch version of Python CDK

clnoll committed:
CDK: upgrade pyarrow (#35818)

ChristoGrab committed:
Source Recurly: bump version to unarchive in Cloud (#35828)

evantahler committed:
docs about how to unarchive a connector (#35825)

clnoll committed:
🤖 Bump patch version of Python CDK

edgao committed:
Destination postgres: upgrade cdk (#35528)

Signed-off-by: Gireesh Sreepathi <gisripa@gmail.com>
Co-authored-by: Gireesh Sreepathi <gisripa@gmail.com>

girarda committed:
Do not build java base image when publishing python cdk

girarda committed:
Revert "Do not build java base image when publishing python cdk"

This reverts commit 82b33c0c6d3362665d99a7dd77465620d9f530af.

girarda committed:
[low-code] allow page size to be defined with string interpolation (#35735)

Co-authored-by: Dan Lecocq <dlecocq@sofi.org>

girarda committed:
🤖 Bump minor version of Python CDK

girarda committed:
🐛 Install arm64 emulator when building source_declarative_manifest (#35839)

girarda committed:
🤖 Bump patch version of Python CDK

rodireich committed:
add configuration for cdc queue size (#35739)

colesnodgrass committed:
Bump Airbyte version from 0.50.54 to 0.51.0

stephane-airbyte committed:
java-CDK fixes to the Junit interceptor (#35827)

evantahler committed:
Un-Archive python destinstions (#35838)

girarda committed:
:bug: low-code: Fix incremental substreams (#35471)

girarda committed:
🤖 Bump patch version of Python CDK

artem1205 committed:
🚨🚨 Source Facebook Marketing: update API to `v19.0` (#35746)

Signed-off-by: Artem Inzhyyants <artem.inzhyyants@gmail.com>

Commit:
✨Source Amazon Ads: migrate source to `YamlDeclarativeSource` with custom `check_connection` (#35481)

Commit:
🏥Source Zendesk Support: fix expected records (#35847)

DanyloGL committed:
Fix tags in metadata files for migrated community connectors  (#35725)

Commit:
📝Source Amazon Seller Partner: update supportLevel to certified (#35848)

clnoll committed:
live-tests: debug mode and initial regression tests framework (#35624)

Co-authored-by: alafanechere <augustin.lafanechere@gmail.com>
Co-authored-by: Augustin <augustin@airbyte.io>

clnoll committed:
Source s3 cache connection (#35808)

maxi297 committed:
Attempt to fix Connector Builder release (#35851)

aaronsteers committed:
Source File: Add Python 3.11 support (#35800)

clnoll committed:
Source S3: upgrade dependencies to avoid transformers transitive dependency (#35857)

stephane-airbyte committed:
get source-mssql CI to green (#35721)

TheAbstractMssqlSourceDatatypeTest was wrong for reals. 
CdcMssqlSourceTest and CdcMssqlSslSourceTest no longer use exclusive containers.
tests that change the SQLServer agent state are now in their own test class (which needs to use an exclusive container)

Besides that, the core of the problem can be grouped in 2 issues:
a) some tests are failing to enable CDC for tables. This is due to a timing issue. We moved that logic into its own function that will try for a total of 240seconds before giving up.
b) some tests are failing when trying to read the minLsn. There is a 1sec wait implemented in the production code. Instead we introduce a busy loop that will wait for a total of 240seconds for records to appear in the CDC table before giving up. That function is implemented in test code.

Unfortunately, for both cases, we sometimes needed to wait while in the middle of a function implemented in the CDK. We introduced a few hooks in the parent PR that are implemented in this PR for the source-mssql tests, and use the functions described above

clnoll committed:
Source Microsoft Onedrive: Bump poetry.lock to upgrade transitive dependency (#35858)

rodireich committed:
make query safer for servers that are case sensitive (#35816)

edgao committed:
Destination Snowflake: Write extracted_at in UTC (#35308)

Signed-off-by: Gireesh Sreepathi <gisripa@gmail.com>
Co-authored-by: Gireesh Sreepathi <gisripa@gmail.com>

ofermend committed:
🐛 Destination Vectara: Improve title extraction logic (#35206)

stephane-airbyte committed:
add background threads to MssqlTestDatabase (#35727)

Commit:
[source-mongodb] record count in state & initial iterator refactor (#35669)

Signed-off-by: Artem Inzhyyants <artem.inzhyyants@gmail.com>
Signed-off-by: Gireesh Sreepathi <gisripa@gmail.com>
Co-authored-by: Augustin <augustin@airbyte.io>
Co-authored-by: Subodh Kant Chaturvedi <subodh1810@gmail.com>
Co-authored-by: Anatolii Yatsuk <35109939+tolik0@users.noreply.github.com>
Co-authored-by: Baz <oleksandr.bazarnov@globallogic.com>
Co-authored-by: Artem Inzhyyants <36314070+artem1205@users.noreply.github.com>
Co-authored-by: Aaron ("AJ") Steers <aj@airbyte.io>
Co-authored-by: Tim Roes <tim@airbyte.io>
Co-authored-by: benmoriceau <benmoriceau@users.noreply.github.com>
Co-authored-by: Gireesh Sreepathi <gisripa@gmail.com>
Co-authored-by: Marius Posta <marius@airbyte.io>
Co-authored-by: Evan Tahler <evan@airbyte.io>
Co-authored-by: Edward Gao <edward.gao@airbyte.io>
Co-authored-by: Anton Karpets <anton.karpets@globallogic.com>
Co-authored-by: Patrick Nilan <nilan.patrick@gmail.com>
Co-authored-by: Akash Kulkarni <113392464+akashkulk@users.noreply.github.com>
Co-authored-by: Tyler B <104733644+tybernstein@users.noreply.github.com>
Co-authored-by: bgroff <bgroff@users.noreply.github.com>
Co-authored-by: mjgatz <86885812+mjgatz@users.noreply.github.com>
Co-authored-by: mgreene <michael.greene@gravie.com>
Co-authored-by: Serhii Lazebnyi <serhii.lazebnyi@globallogic.com>
Co-authored-by: Serhii Lazebnyi <53845333+lazebnyi@users.noreply.github.com>
Co-authored-by: Rodi Reich Zilberman <867491+rodireich@users.noreply.github.com>
Co-authored-by: Daryna Ishchenko <80129833+darynaishchenko@users.noreply.github.com>
Co-authored-by: Stephane Geneix <147216312+stephane-airbyte@users.noreply.github.com>
Co-authored-by: Joe Reuter <joe@airbyte.io>
Co-authored-by: Marcos Marx <marcosmarxm@users.noreply.github.com>
Co-authored-by: Maxime Carbonneau-Leclerc <3360483+maxi297@users.noreply.github.com>
Co-authored-by: Akash Kulkarni <akash@airbyte.io>
Co-authored-by: Roman Yermilov [GL] <86300758+roman-yermilov-gl@users.noreply.github.com>
Co-authored-by: Alexandre Girard <alexandre@airbyte.io>
Co-authored-by: girarda <girarda@users.noreply.github.com>
Co-authored-by: Brian Lai <51336873+brianjlai@users.noreply.github.com>
Co-authored-by: brianjlai <brianjlai@users.noreply.github.com>
Co-authored-by: Catherine Noll <clnoll@users.noreply.github.com>
Co-authored-by: midavadim <midavadim@yahoo.com>
Co-authored-by: Julien COUTAND <julien.coutand@gmail.com>
Co-authored-by: Christo Grabowski <108154848+ChristoGrab@users.noreply.github.com>
Co-authored-by: maxi297 <maxime@airbyte.io>
Co-authored-by: Bindi Pankhudi <bindi@airbyte.io>
Co-authored-by: Bindi Pankhudi <bindi@airbyte.com>
Co-authored-by: Ben Drucker <bvdrucker@gmail.com>
Co-authored-by: TornadoContre <37258495+TornadoContre@users.noreply.github.com>
Co-authored-by: Natik Gadzhi <natik@respawn.io>
Co-authored-by: Thomas Dippel <dipth@users.noreply.github.com>
Co-authored-by: marcosmarxm <marcosmarxm@gmail.com>
Co-authored-by: Alex Birdsall <ambirdsall@gmail.com>
Co-authored-by: ambirdsall <ambirdsall@users.noreply.github.com>
Co-authored-by: Jose Gerardo Pineda <jose.pineda@airbyte.io>
Co-authored-by: alafanechere <augustin.lafanechere@gmail.com>
Co-authored-by: Pedro S. Lopez <pedroslopez@me.com>
Co-authored-by: Ella Rohm-Ensing <erohmensing@gmail.com>
Co-authored-by: Siarhei Ivanou <sinusu@gmail.com>
Co-authored-by: Anatolii Yatsuk <tolikyatsuk@gmail.com>
Co-authored-by: Ryan Waskewich <156025126+rwask@users.noreply.github.com>
Co-authored-by: Sajarin <sajarindider@gmail.com>
Co-authored-by: artem1205 <artem1205@users.noreply.github.com>
Co-authored-by: perangel <perangel@gmail.com>
Co-authored-by: Joe Bell <joseph.bell@airbyte.io>
Co-authored-by: Obioma Anomnachi <onanomnachi@gmail.com>
Co-authored-by: maxi297 <maxi297@users.noreply.github.com>
Co-authored-by: SatishChGit <satishchinthanippu@gmail.com>
Co-authored-by: Brian Leonard <brian@bleonard.com>
Co-authored-by: David Wallace <dwallace0723@gmail.com>
Co-authored-by: pmossman <pmossman@users.noreply.github.com>
Co-authored-by: Stephane Geneix <stephane@airbyte.io>
Co-authored-by: Alexandre Cuoci <Hesperide@users.noreply.github.com>
Co-authored-by: Danny Tiesling <tiesling@gmail.com>

ambirdsall committed:
Pass docker hub credentials to airbyte-ci's `bump_version` command (#35802)

akashkulk committed:
[Source-postgres]: Add logging in case of multiple records with same LSN (#35842)

natikgadzhi committed:
Docs: Tutorials formatting + from scratch connector tutorial cleanup (#33839)

Co-authored-by: Marcos Marx <marcosmarxm@users.noreply.github.com>

clnoll committed:
Source Instagram: remove "total_interactions" from MediaInsights queries (#35875)

tolik0 committed:
:bug: Source S3: Return iam docs (#35850)

alafanechere committed:
Fix tags in metadata.yaml files: declare cdk tags (#35670)

darynaishchenko committed:
:sparkles: Source Bing Ads: new streams: Audience Performance Report, Goals And Funnels Report, Product Dimension Performance Report (#35812)

alafanechere committed:
live-tests: add regression tests suite (#35837)

stephane-airbyte committed:
source-mssql: reduce the number of containers used in tests (#35855)

stephane-airbyte committed:
speed up source-mssql tests (#35799)

pnilan committed:
Source Metabase: Updates `dashboards` stream, migrates to poetry (#35680)

maxi297 committed:
Declarative CDK: Fix None error on stream_slice (#35879)

maxi297 committed:
🤖 Bump patch version of Python CDK

girarda committed:
🐛 follow up to #35471: update the cartesian stream slicer (#35865)

girarda committed:
🤖 Bump patch version of Python CDK

Anush008 committed:
📝 Updated Qdrant destination icon (#35104)

Co-authored-by: Marcos Marx <marcosmarxm@users.noreply.github.com>
Co-authored-by: marcosmarxm <marcosmarxm@gmail.com>

shrutimantri committed:
Source Ringcentral: add logo (#34242)

Co-authored-by: Marcos Marx <marcosmarxm@users.noreply.github.com>
Co-authored-by: marcosmarxm <marcosmarxm@gmail.com>

Commit:
[source-postgres] State counter on postgres (#34724)

Co-authored-by: Aaron ("AJ") Steers <aj@airbyte.io>
Co-authored-by: Joe Bell <joseph.bell@airbyte.io>
Co-authored-by: Marcos Marx <marcosmarxm@users.noreply.github.com>
Co-authored-by: SatishChGit <satishchinthanippu@gmail.com>
Co-authored-by: evantahler <evan@airbyte.io>
Co-authored-by: Rodi Reich Zilberman <867491+rodireich@users.noreply.github.com>
Co-authored-by: Joe Reuter <joe@airbyte.io>
Co-authored-by: Catherine Noll <clnoll@users.noreply.github.com>
Co-authored-by: Anton Karpets <anton.karpets@globallogic.com>
Co-authored-by: Christo Grabowski <108154848+ChristoGrab@users.noreply.github.com>
Co-authored-by: Akash Kulkarni <akash@airbyte.io>
Co-authored-by: Akash Kulkarni <113392464+akashkulk@users.noreply.github.com>
Co-authored-by: Gireesh Sreepathi <gisripa@gmail.com>
Co-authored-by: Artem Inzhyyants <36314070+artem1205@users.noreply.github.com>
Co-authored-by: Ella Rohm-Ensing <erohmensing@gmail.com>
Co-authored-by: Daryna Ishchenko <80129833+darynaishchenko@users.noreply.github.com>
Co-authored-by: Baz <oleksandr.bazarnov@globallogic.com>
Co-authored-by: Patrick Nilan <nilan.patrick@gmail.com>
Co-authored-by: Serhii Lazebnyi <53845333+lazebnyi@users.noreply.github.com>
Co-authored-by: Marius Posta <marius@airbyte.io>
Co-authored-by: Chandler Prall <chandler.prall@gmail.com>
Co-authored-by: pmossman <pmossman@users.noreply.github.com>
Co-authored-by: Anatolii Yatsuk <35109939+tolik0@users.noreply.github.com>
Co-authored-by: Cole Snodgrass <cole@airbyte.io>
Co-authored-by: Anatolii Yatsuk <tolikyatsuk@gmail.com>
Co-authored-by: bgroff <bgroff@users.noreply.github.com>
Co-authored-by: Maxime Carbonneau-Leclerc <3360483+maxi297@users.noreply.github.com>
Co-authored-by: Ryan Waskewich <156025126+rwask@users.noreply.github.com>
Co-authored-by: maxi297 <maxi297@users.noreply.github.com>
Co-authored-by: Pedro S. Lopez <pedroslopez@me.com>
Co-authored-by: Håkon Åmdal <hakon@aamdal.com>
Co-authored-by: Roman Yermilov [GL] <86300758+roman-yermilov-gl@users.noreply.github.com>
Co-authored-by: Augustin <augustin@airbyte.io>
Co-authored-by: Lake Mossman <lake@airbyte.io>
Co-authored-by: lmossman <lmossman@users.noreply.github.com>
Co-authored-by: Brian Lai <51336873+brianjlai@users.noreply.github.com>
Co-authored-by: Subodh Kant Chaturvedi <subodh1810@gmail.com>
Co-authored-by: Stephane Geneix <147216312+stephane-airbyte@users.noreply.github.com>
Co-authored-by: Sajarin <sajarindider@gmail.com>
Co-authored-by: Edward Gao <edward.gao@airbyte.io>
Co-authored-by: nguyenaiden <duy@airbyte.io>
Co-authored-by: Natalie Kwong <38087517+nataliekwong@users.noreply.github.com>
Co-authored-by: terencecho <3916587+terencecho@users.noreply.github.com>
Co-authored-by: Alex Birdsall <ambirdsall@gmail.com>

ChristoGrab committed:
Docs: Fix link in Core Concepts page (#35900)

gisripa committed:
Destination Snowflake: Handle NPE for state value.  (#35899)

Hesperide committed:
updated instructions for external logging (#35903)

rodireich committed:
GA release 4.0.0 (#35873)

alafanechere committed:
connectors-qa: check cdk and language tags (#35685)

artem1205 committed:
🚨🚨 Source Hubspot: Update `Deals Property History` and `Companies Property History` to API `v3` (#35662)

Signed-off-by: Artem Inzhyyants <artem.inzhyyants@gmail.com>

maxi297 committed:
Attempt to fix transient nightly build errors: Remove poetry cache (#35894)

marcosmarxm committed:
🐛 Source Orb: update enrich ledger entry with event stream to pass timeframe bounds (#35897)

Co-authored-by: Kshitij Grover <k.grover123@gmail.com>
Co-authored-by: Arsh <arsh@withorb.com>

artem1205 committed:
Source GitHub: update CDK (#35915)

Signed-off-by: Artem Inzhyyants <artem.inzhyyants@gmail.com>

artem1205 committed:
Source Facebook Marketing: Fix lookback window (#35913)

Signed-off-by: Artem Inzhyyants <artem.inzhyyants@gmail.com>

Commit:
.github: remove unused connector_dependency_template.md (#35917)

Commit:
delete unused airbyte-base-java-image top-level folder (#35918)

terencecho committed:
Bump Airbyte version from 0.51.0 to 0.52.0

ChristoGrab committed:
Source Google Search Console: Update records/test config (#35927)

Commit:
.github: clean up CODEOWNERS (#35919)

Commit:
.github: delete pull_request_template.md (#35921)

erohmensing committed:
Introduce FinalStateCursor to emit state messages at the end of full refresh syncs (#35905)

Co-authored-by: brianjlai <brian.lai@airbyte.io>

erohmensing committed:
CDK: assert >0 state messages per read (fix tests) (#35906)

<!--
Thanks for your contribution! 
Before you submit the pull request, 
I'd like to kindly remind you to take a moment and read through our guidelines
to ensure that your contribution aligns with the type of contributions our project accepts.
All the information you need can be found here:
   https://docs.airbyte.com/contributing-to-airbyte/

We truly appreciate your interest in contributing to Airbyte,
and we're excited to see what you have to offer! 

If you have any questions or need any assistance, feel free to reach out in #contributions Slack channel.
-->

## What
* After https://github.com/airbytehq/airbyte/pull/35905, we should be emitting a state message with every successful sync. However there are a few tests that were too lenient and weren't actually _successful_ syncs. This PR fixes those cases and adds validation that we emit at least one state message per successful sync. 

## How
* Add an assertion that we get at least 1 state message for a successful sync 
* Fix some tests that previously "output 0 expected records" but actually errored silently - do not run them as read tests
* Fix a test that failed silently due to lack of support for multi-format
* Add a new test for syncs that output 0 records successfully

## 🚨 User Impact 🚨
None - test changes


## Pre-merge Actions
*Expand the relevant checklist and delete the others.*

<details><summary><strong>New Connector</strong></summary>

### Community member or Airbyter

- **Community member?** Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- Connector version is set to `0.0.1`
    - `Dockerfile` has version `0.0.1`
- Documentation updated
    - Connector's `README.md`
    - Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - `docs/integrations/<source or destination>/<name>.md` including changelog with an entry for the initial version. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
    - `docs/integrations/README.md`

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- Create a non-forked branch based on this PR and test the below items on it
- Build is successful
- If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).

</details>

<details><summary><strong>Updating a connector</strong></summary>

### Community member or Airbyter

- Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- Unit & integration tests added


### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- Create a non-forked branch based on this PR and test the below items on it
- Build is successful
- If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).

</details>

<details><summary><strong>Connector Generator</strong></summary>

- Issue acceptance criteria met
- PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/resources/pull-requests-handbook)
- If adding a new generator, add it to the [list of scaffold modules being tested](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connector-templates/generator/build.gradle#L41)
- The generator test modules (all connectors with `-scaffold` in their name) have been updated with the latest scaffold by running `./gradlew :airbyte-integrations:connector-templates:generator:generateScaffolds` then checking in your changes
- Documentation which references the generator is updated as needed

</details>

<details><summary><strong>Updating the Python CDK</strong></summary>

### Airbyter

Before merging:
- Pull Request description explains what problem it is solving
- Code change is unit tested
- Build and my-py check pass
- Smoke test the change on at least one affected connector
   - On Github: Run [this workflow](https://github.com/airbytehq/airbyte/actions/workflows/connectors_tests.yml), passing `--use-local-cdk --name=source-<connector>` as options
   - Locally: `airbyte-ci connectors --use-local-cdk --name=source-<connector> test`
- PR is reviewed and approved
      
After merging:
- [Publish the CDK](https://github.com/airbytehq/airbyte/actions/workflows/publish-cdk-command-manually.yml)
   - The CDK does not follow proper semantic versioning. Choose minor if this the change has significant user impact or is a breaking change. Choose patch otherwise.
   - Write a thoughtful changelog message so we know what was updated.
- Merge the platform PR that was auto-created for updating the Connector Builder's CDK version
   - This step is optional if the change does not affect the connector builder or declarative connectors.

</details>

davinchia committed:
Better Heartbeating External Documentation. (#35932)

In,
- airbytehq/airbyte-platform-internal@96baf5b
- Better Destination Heartbeat Error Messages airbyte-platform-internal#11595

we improve our heartbeat error messages and point users to this external document.

Here, we improve external documentation to help users understand what is happening and what they can do.

davinchia committed:
Improve Heartbeating External Docs. (#35934)

Better explanation for:

General philosophy.
What a heartbeat error means.

erohmensing committed:
Emit record counts in state messages for concurrent streams (#35907)

Co-authored-by: brianjlai <brian.lai@airbyte.io>
Co-authored-by: Brian Lai <51336873+brianjlai@users.noreply.github.com>

brianjlai committed:
🤖 Bump minor version of Python CDK

bindipankhudi committed:
added integration instructions to getting started (#35861)

Co-authored-by: bindipankhudi <bindi@airbyte.com>

alafanechere committed:
live-tests: pass connection id (#35922)

alafanechere committed:
live-tests: add duckdb backend (#35923)

tolik0 committed:
📝 Source Amazon Ads: Update tags in metadata.yaml (#35951)

rwask committed:
Update on-kubernetes-via-helm.md - 0.52.0 helm charts upgrade informa… (#35935)

Commit:
🐛 Source Shopify: Fixed bug when `start date` was not provided, but the stream was using it (#35952)

clnoll committed:
Source S3: pin `transformers` transitive dependency (#35955)

terencecho committed:
Update enterprise setup guide for storage (#35966)

nataliekwong committed:
[Docs] Edit recharge wording (#35975)

akashkulk committed:
[source-postgres] : Add retries back to the DBZ connector (#35904)

brianjlai committed:
[airbyte-cdk] entrypoint wrapper should use per-stream state not legacy format (#35976)

brianjlai committed:
🤖 Bump patch version of Python CDK

clnoll committed:
Source Microsoft Onedrive: pin transformers transitive dependency (#35956)

stephane-airbyte committed:
fix junit versionning mess (#35977)

alafanechere committed:
migrate live-test to a different repo (#35948)

Commit:
✨Source Amazon Seller Partner: add `GET_VENDOR_FORECASTING_REPORT` streams (#35954)

tolik0 committed:
🏥Source Mixpanel: fix expected records (#35964)

Commit:
🐛 Source Recharge: Add `sort_by` to guarantee the records are in `ASC` order (#35982)

ChristoGrab committed:
Source Amplitude: unpin CDK (#35987)

girarda committed:
raise exception with the full class name if a class for a custom comp… (#35868)

girarda committed:
🤖 Bump patch version of Python CDK

ChristoGrab committed:
Source Zoom: Fix authenticator bug and add missing fields (#35369)

erohmensing committed:
🤖 Bump minor version of Python CDK

artem1205 committed:
🐛 Source Github: Handle rate limit exception as config error (#35986)

Signed-off-by: Artem Inzhyyants <artem.inzhyyants@gmail.com>

ChristoGrab committed:
Disable Google Workspace Admin Reports in registries (#35965)

ChristoGrab committed:
Archive Google Workspace Admin Reports (#35967)

brianjlai committed:
🐛 [source-shopify] Fix discover command if shop name does not exist (#36000)

Commit:
.editorconfig: add kotlin rules which approximate airbyte-ci format (#36017)

Commit:
gradle: add kotlin support in all projects, warnings as errors (#36018)

blarghmatey committed:
[airbyte-cdk] Fix tab delimiter configuration in CSV file type (#35901)

amaliaroye committed:
Update linkedin-ads.md (#36030)

rodireich committed:
configure debezium to correctly convert column with default value whe… (#36011)

yvonnegitau committed:
chore: format code

yvonnegitau committed:
chore: format code

yvonnegitau committed:
made changes to docs

yvonnegitau committed:
made changes to docs

yvonnegitau committed:
made changes to docs

yvonnegitau committed:
made changes to docs

brianjlai committed:
🤖 Bump patch version of Python CDK

aaronsteers committed:
Docs (PyAirbyte): Add pyairbyte telemetry information and top-level `/telemetry` docs redirect (#36034)

pmossman committed:
Bump Airbyte version from 0.52.0 to 0.52.1

akashkulk committed:
[Source-postgres] : Compare each record’s lsn_commit value instead of lsn_proc. (#35939)

marcosmarxm committed:
🐛 Source Orb: fix window request for `credit_ledger_entries` (#36036)

pnilan committed:
Source Chargebee - Implement integration testing for otherwise untested streams [ITAS] (#35509)

gisripa committed:
Destinations CDK: Plumbing related to airbyte_meta from protocol to raw table (#35944)

Co-authored-by: Edward Gao <edward.gao@airbyte.io>

Hesperide committed:
Install Command Update, External Secret Management & Kubernetes Secrets (#35999)

Hesperide committed:
fix sso auth block (#36038)

Commit:
[source-mssql] mssql use SourceStateIterator to send out counts (#35792)

Signed-off-by: Gireesh Sreepathi <gisripa@gmail.com>
Signed-off-by: Artem Inzhyyants <artem.inzhyyants@gmail.com>
Co-authored-by: Stephane Geneix <147216312+stephane-airbyte@users.noreply.github.com>
Co-authored-by: Anton Karpets <anton.karpets@globallogic.com>
Co-authored-by: Daryna Ishchenko <80129833+darynaishchenko@users.noreply.github.com>
Co-authored-by: Anatolii Yatsuk <35109939+tolik0@users.noreply.github.com>
Co-authored-by: Christo Grabowski <108154848+ChristoGrab@users.noreply.github.com>
Co-authored-by: Evan Tahler <evan@airbyte.io>
Co-authored-by: Gireesh Sreepathi <gisripa@gmail.com>
Co-authored-by: Edward Gao <edward.gao@airbyte.io>
Co-authored-by: Joe Bell <joseph.bell@airbyte.io>
Co-authored-by: Augustin <augustin@airbyte.io>
Co-authored-by: Artem Inzhyyants <36314070+artem1205@users.noreply.github.com>
Co-authored-by: Marius Posta <marius@airbyte.io>
Co-authored-by: Brian Lai <51336873+brianjlai@users.noreply.github.com>
Co-authored-by: maxi297 <maxime@airbyte.io>
Co-authored-by: Ella Rohm-Ensing <erohmensing@gmail.com>
Co-authored-by: erohmensing <erohmensing@users.noreply.github.com>
Co-authored-by: Catherine Noll <clnoll@users.noreply.github.com>
Co-authored-by: Alexandre Girard <alexandre@airbyte.io>
Co-authored-by: Dan Lecocq <dlecocq@sofi.org>
Co-authored-by: girarda <girarda@users.noreply.github.com>
Co-authored-by: colesnodgrass <colesnodgrass@users.noreply.github.com>
Co-authored-by: Danylo Jablonski <150933663+DanyloGL@users.noreply.github.com>
Co-authored-by: alafanechere <augustin.lafanechere@gmail.com>
Co-authored-by: Maxime Carbonneau-Leclerc <3360483+maxi297@users.noreply.github.com>
Co-authored-by: Aaron ("AJ") Steers <aj@airbyte.io>
Co-authored-by: Rodi Reich Zilberman <867491+rodireich@users.noreply.github.com>
Co-authored-by: Ofer Mendelevitch <ofermend@gmail.com>

colesnodgrass committed:
Bump Airbyte version from 0.52.1 to 0.53.0

natikgadzhi committed:
Updated connector development overview doc (#36010)

yvonnegitau committed:
QA is failing

yvonnegitau committed:
QA is failing

yvonnegitau committed:
QA is failing

yvonnegitau committed:
QA is failing

yvonnegitau committed:
QA is failing

yvonnegitau committed:
QA is failing

yvonnegitau committed:
QA is failing

yvonnegitau committed:
QA is failing

rwask committed:
Update on-kubernetes-via-helm.md (#35997)

ChristoGrab committed:
Source Greenhouse: unpin CDK (#35988)

gisripa committed:
Destination Redshift:  Adopt CDK changes for `airbyte_meta` (#35981)

stephane-airbyte committed:
fix connector changelogs (#36039)

As we're trying to automate changelog editing, we need to make sure it's following a common format. Most connectors are OK except the ones here.
Issus include:
Missing a line before the changelog table (as required per markdown)
missing a header separator (as required by makdown)
having weird characters in the header separator
Wrong headers
missing column

stephane-airbyte committed:
extract out the connector changelog modification out of the bump_version code (#34586)

We want to move the changelog mutation out of the connectors command, so we can reuse it for java CDK.
all connectors were tested, and the only ones that were failing are fixed in #36039

marcosmarxm committed:
Update Issue Templates (#35896)

pnilan committed:
Source Chargebee: Declare Unexpected Fields (#36037)

Commit:
java CDK: add static deserialize*(byte[]) methods to Jsons (#36019)

These aren't used anywhere in this PR stack, but it's dumb that these methods don't exist. I want to ship this change in this PR stack's CDK release.

Commit:
java-cdk: prettier gradle dependency ordering (#36020)

Commit:
java-cdk: move log4j2-test.xml to test fixtures (#36021)

Commit:
java-cdk: change log level in log4j2-test.xml (#36023)

erohmensing committed:
source-zoom: reference published version in breaking changes (#36135)

blarghmatey committed:
[source-s3] Bump python-cdk version to include CSV tab delimiter fix (#36160)

jbfbell committed:
update redshift docs (#36041)

nataliekwong committed:
[Docs] Release Notes for February 2024 (#35909)

Hesperide committed:
Add Enterprise API instructions (#36172)

Co-authored-by: Bryce Groff <bgroff@hawaii.edu>

subodh1810 committed:
upgrade launch darkly sdk to version 7.2.6 (#36013)

Commit:
🎉 Source PayPal Transactions: Extend `Disputes` stream schema with missing properties (#36173)

Commit:
Convert Async Destination Framework to Kotlin (#36033)

Commit:
docs: fix or remove broken links (#36185)

Commit:
🐛 Source Shopify: fix `STATE` messages frequency for the `nested` sub-streams (#36170)

girarda committed:
🐛 low-code: remove debug sleep (#36187)

girarda committed:
🤖 Bump patch version of Python CDK

evantahler committed:
fix spacing and bad alerts (#36188)

Commit:
gradle: build cache tweaks (#36055)

Commit:
java-cdk: update README (#36191)

ChristoGrab committed:
Source Paypal Transaction: Unpin CDK (#36165)

Commit:
connector-templates: delete source-singer template (#36193)

Commit:
delete bad or useless README files (#36196)

Commit:
gradle: remove unused pmd rules (#36195)

Commit:
Wrap default namespace in optional to avoid NPE (#36207)

Commit:
docs: update and remove some bad content (#36197)

Commit:
delete unused airbyte-integrations/infrastructure terraform stuff (#36194)

ambirdsall committed:
Base datetime cursor state off latest observed record (#35843)

lmossman committed:
Bump Airbyte version from 0.53.0 to 0.53.1

girarda committed:
Add PR review check for community-contractor pull requests (#36166)

ambirdsall committed:
🤖 Bump minor version of Python CDK

brianjlai committed:
✨ [source-facebook-marketing] [source-google-analytics-data-api] [source-salesforce] [source-stripe] bumping airbyte-cdk, regenerating poetry lock and fixing various tests (#35978)

Commit:
🎉 Source Google Ads: Add `Unrecognized Field` description while `check_connection` (#36208)

lazebnyi committed:
CAT: add validation for state messages (#36001)

Commit:
🐛 Source Intercom: Fixed bug causing the `header` values to set incorrectly (#36232)

tolik0 committed:
🏥 Source Twilio: Update expected records (#36228)

tolik0 committed:
✨ Source Microsoft SharePoint: Add parsing shared files (#35830)

tolik0 committed:
🚨🚨✨ Source Facebook Pages: Upgrade Facebook API to v19.0 (#36015)

Co-authored-by: airt <airtial@gmail.com>

girarda committed:
low-code: Allow developers to use a custom schema loader (#36040)

subodh1810 committed:
java-cdk-destination: mark logs as debug (#36227)

girarda committed:
🤖 Bump minor version of Python CDK

artem1205 committed:
Airbyte CDK (low code): add refresh_token_error handler to `DeclarativeOauth2Authenticator` (#36058)

Signed-off-by: Artem Inzhyyants <artem.inzhyyants@gmail.com>
Co-authored-by: Alexandre Girard <alexandre@airbyte.io>

artem1205 committed:
🤖 Bump patch version of Python CDK

maxi297 committed:
Revert "Attempt to fix transient nightly build errors: Remove poetry … (#36206)

gisripa committed:
Destination Redshift: Json transformer for graceful handling of large records (#36203)

This includes CDK changes to wire the `Transformer` interface into `AsyncStreamConsumer`

pedroslopez committed:
Add extensibility team to python connector approvers (#36248)

ChristoGrab committed:
Source Typeform: Unpin CDK version (#36164)

evantahler committed:
Update supported-data-types and nullability (#36259)

ambirdsall committed:
Remove `most_recent_record` arg from `Cursor.close_slice` (#36216)

rodireich committed:
sanity

rodireich committed:
Revert "sanity"

This reverts commit 5533eed8cbbf30e3498206469ef66dbc4ee365b7.

Commit:
remove infracost files

d-kimanthi committed:
remove infracost files

Commit:
rebase to master

Commit:
rebase to master

amaliaroye committed:
Update bing-ads.md (#36279)

amaliaroye committed:
Update bing-ads.md (#36279)

Commit:
merge with remote

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/airbytehq/airbyte?pullRequest=36280) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/airbytehq/airbyte?pullRequest=36280) before we can accept your contribution.<br/>**14** out of **15** committers have signed the CLA.<br/><br/>:white_check_mark: tolik0<br/>:white_check_mark: girarda<br/>:white_check_mark: subodh1810<br/>:white_check_mark: artem1205<br/>:white_check_mark: maxi297<br/>:white_check_mark: gisripa<br/>:white_check_mark: pedroslopez<br/>:white_check_mark: evantahler<br/>:white_check_mark: ChristoGrab<br/>:white_check_mark: yvonnegitau<br/>:white_check_mark: rodireich<br/>:white_check_mark: amaliaroye<br/>:white_check_mark: ambirdsall<br/>:white_check_mark: d-kimanthi<br/>:x: Kimanthi<br/><hr/>**Kimanthi** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/airbytehq/airbyte?pullRequest=36280) it.</sub>

Commit:
update change log

d-kimanthi committed:
update change log

Commit:
Merge branch 'master' into source-surveycto-add-streams

d-kimanthi committed:
update docs

d-kimanthi committed:
update docs

marcosmarxm commented:
@d-jeph need all contributors to sign the CLA. Can you ask them to do it?

d-kimanthi committed:
made changes to docs

d-kimanthi committed:
made changes to docs

d-kimanthi committed:
update author

d-kimanthi committed:
fix conflict

d-kimanthi committed:
resolve conflicts

d-kimanthi committed:
fix conflicts

d-kimanthi committed:
resolve conflicts

d-kimanthi committed:
fix mc

d-kimanthi committed:
# This is a combination of 3 commits.
# This is the 1st commit message:

fix mc

# This is the commit message #2:

QA is failing

# This is the commit message #3:

QA is failing

d-kimanthi committed:
fix mc

QA is failing

QA is failing

rebase to master

remove infracost files

squash commits

d-kimanthi committed:
fix mc

d-kimanthi committed:
fix mc

marcosmarxm commented:
Still missing 1 contributor 📦

d-kimanthi committed:
update readme

d-jeph commented:
Closing to recreate PR properly.
